app/mailers/notifier.rb

```diff
@@ -5,12 +5,12 @@ def self.report_load_event(schema, event)
     }
   end
 
-  def send_msg(schema, email_addr, subject, body)
+     def send_msg(schema, email_addr, subject, event)
```

Comment:
1. fix indentation here
2. remove the schema as a parameter
3. Update all the calls to this method. Do a search in the project for `send_msg` you will need to remove `schema` and change the `body` to `event`